docs/how-to-translate-files.md

```diff
@@ -1,15 +1,79 @@
+
+## Prepare yourself for contributions
+
+> The freeCodeCamp Localization Roadmap – There Are No Speed Limits
+
+You can translate as much as you want, when you want. It's only a matter of how much time and energy you are willing to invest as a volunteer translator.
+
+Our ask is that you understand the below:
+
+1. **Translations are a team effort.**
+
+   Translating freeCodeCamp's resources is one of the most fun and rewarding experiences as a contributor, and it works only if you get your friends and colleagues who speak the same world language as you involved.
+
+   We recommend joining [our community forum](https://forum.freecodecamp.org/c/contributors/3) and [translators chat room](https://chat.freecodecamp.org/channel/translators) with your friends and showing your interest before starting off with translations. Contributing translations are relatively trivial however, it's also a lot of work.
+
+   We want you enjoy contributing and not burn out or lose interest.
+
+   A small group of 4-5 individuals is a good size to start your niche for your world language. You can then recruit even more friends to join the team.
 
-To begin, head to our translation website and login (if you have not contributed to translations before, you will need to create an account).
+2. **It costs quite a lot to spin servers for each language.**
+
+   On the surface it might not seem how complicated the technical stack is, but it costs quite a lot to keep the engines running. This includes provisioning additional servers and dedicating staff to look after them.
+
+   freeCodeCamp.org is committed to providing these for free as always, however we need to prioritize resources for those who need it the most. The last thing we want is to shutdown servers for a language if the translation activity dies off & things become outdated.
+
+   Once a language reaches at least a few certifications on the curriculum we can begin deploying the language live on [`/learn`](https://www.freecodecamp.org/learn), while you continue to translate the remaining certifications.
+
+   For example, we would want to deploy the at least the entire front-end certifications suite when we ship a new world language for the first time.
+
+3. **But what about the languages not listed on the translation platform?**
+
+   We have looked at our user base and added 30+ most widely spoken languages to the list of enabled languages on the translations platform. Some languages like Chinese and Spanish are already deployed live on **"/learn"** at this moment.
+
+   Unfortunately, the list does not include hundreds of languages out there. We get dozens of requests from contributors like you every day to enable a language they speak and would like to help translate to.
+
+   We are definitely looking forward to adding more languages to the list, but as you may already guess, it would only be feasible if we get enough momentum and around a world language.
```

Comment:
```suggestion
   We are definitely looking forward to adding more languages to the list, but as you may already guess, it would only be feasible if we get enough momentum around a world language.
```